src/manage.c

```diff
@@ -230,13 +282,19 @@ void manage_acq()
   acq_send_result(prn, snr, cp, cf);
   if (snr < ACQ_THRESHOLD) {
     /* Didn't find the satellite :( */
-    acq_prn_param[prn].state = ACQ_PRN_TRIED;
+    float dilute = (acq_prn_param[prn].dopp_hint_high -
```

Comment:
Add a comment to explain what `dilute` is trying to do, i.e. double doppler search space